### PR TITLE
make the description field mandatory 

### PR DESCRIPTION
**Pull request by shivam333rawat**
Screenshot 2022-08-26 125511
Targets #2254

### Description
Make the description field mandatory on recurring sales Expenses

### Checklist:

- [x] I have performed a self-review of my own code.

### CODE REVIEW

File: Modules/Expense/Resources/views/recurring/edit.blade.php [hunk 1]
```diff
@@ -83,7 +83,7 @@
                                 </div>
 
                                 <div class="form-group">
-                                    <label for="description" class="">Description</label>
+                                    <label for="description" class="field-required">Description</label>
```
Comment (gautam071) [inline Modules/Expense/Resources/views/recurring/edit.blade.php:86]:
```suggestion
                                    <label for="description" required="required">Description</label>
```

**Review by gautam071 — APPROVED** [review]
Looks good one suggestion

File: Modules/Expense/Resources/views/recurring/edit.blade.php [hunk 2]
```diff
@@ -83,7 +83,7 @@
                                 </div>
 
                                 <div class="form-group">
-                                    <label for="description" class="">Description</label>
+                                    <label for="description" required = "required">Description</label>
```
Comment (P4NK4J) [inline Modules/Expense/Resources/views/recurring/edit.blade.php:86]:
```suggestion
                                    <label for="description" required>Description</label>
```

How about this ?

**Review by P4NK4J — COMMENTED** [review]
Left a question for you

**Review by P4NK4J — APPROVED** [review]
Looks good.

**Comment by P4NK4J** [comment]
@shivam333rawat I think you need to add `*` as well, as per the issue @gautam071 You can confirm this, but having a look at the issue description, it seems like a red * would be required.

**Comment by gautam071** [comment]
@P4NK4J You are right.

File: Modules/Expense/Resources/views/recurring/edit.blade.php [hunk 3]
```diff
@@ -83,8 +83,8 @@
                                 </div>
 
                                 <div class="form-group">
-                                    <label for="description" class="">Description</label>
-                                    <textarea name="description" id="description" rows="5" class="form-control">{{ $recurringExpense->description }}</textarea>
+                                    <label for="description" class="field-required">Description</label>
```
Comment (gautam071) [inline Modules/Expense/Resources/views/recurring/edit.blade.php:86]: Screenshot 2022-08-25 at 10 29 41 PM

Add the red astrix similar to the image

**Review by gautam071 — CHANGES_REQUESTED** [review]

**Review by gautam071 — APPROVED** [review]

**Pull request merged**